### ISSUE TITLE
If 30,20,10 seconds to reaction

### ISSUE THREAD

**Issue by HeCosmin** [johnschult/jquery.countdown360#30]
Example like this [codepen](https://codepen.io/geoffgraham/pen/yLywVbW)

30 seconds - primary
20 seconds - warning
10 seconds - danger

I tried trying but doesn't work I don't understand where your language exactly function. example like this

			if(countdown.seconds <= "10")
			    {
					alert("test");
				}
				
Help me bit out this. Thank you very much!

**Comment by johnschult** [comment]
Yeah, I am not really maintaining this any longer. I don't use jquery for anything and have very little interest. If you or someone else is interested in taking over and adding new features then speak up 😀

**Comment by HeCosmin** [comment]
I know this no longer use but your script better works fine all functions
than other has issue function doesn't work, yes better add features if else
function from count seconds change colours or custom class. I hope come
back 😄♥️

Il dom 14 nov 2021, 23:52 John Schult ***@***.***> ha scritto:
